plugin/code_lens.py

```diff
@@ -131,10 +133,36 @@ def resolved_code_lens(self) -> Iterable[CodeLensData]:
             if lens.is_resolved():
                 yield lens
 
-    def render(self) -> None:
-        accent = self.view.style_for_scope("region.greenish markup.accent.codelens.lsp")["foreground"]
-        for index, lens in enumerate(self._flat_iteration()):
-            self.view.add_regions(self._region_key(index), [lens.region], "", "", 0, [lens.small_html], accent)
+    def _get_phantom_region(self, region: sublime.Region) -> sublime.Region:
+        """Takes the current region and puts the phantom on the line above, properly indented.
+
+        Code-lenses are typically shown above the code in VSCode.
+        """
```

Comment:
Nits:
- this doesn't strictly put the phantom anywhere, just creates a region
- Not sure if the note about VSCode is necessary in that context.